scene/gui/graph_node.cpp

```diff
@@ -193,13 +194,15 @@ void GraphNode::_notification(int p_what) {
 
 	if (p_what == NOTIFICATION_DRAW) {
 
+		Ref<StyleBox> sb_h;
 		Ref<StyleBox> sb;
 
 		if (comment) {
+			sb_h = get_stylebox(selected ? "commentfocus_h" : "comment_h");
 			sb = get_stylebox(selected ? "commentfocus" : "comment");
 
 		} else {
-
+			sb_h = get_stylebox(selected ? "selectedframe_h" : "frame_h");
```

Comment:
The theme name should be more descriptive. call it "frame_header"

Reply:
Wow, your version looks more clear than mine. Why you dont PR this ?